locale/fr_CA/admin.xml

```diff
@@ -54,7 +54,7 @@
 	<message key="admin.mergeUsers.into.description">Sélectionner un utilisateur à qui attribuer la paternité des textes, les assignations de révisions etc. de l'utilisateur précédent.</message>
 	<message key="admin.mergeUsers.from.description">Sélectionner un utilisateur à fusionner au compte d'un autre utilisateur (par ex. : lorsque quelqu'un possède deux comptes utilisateur). Le compte sélectionné en premier sera supprimé et ses soumissions, assignations, etc. seront affectées au second compte.</message>
 	<message key="admin.mergeUsers.allUsers">Tous les utilisateurs inscrits à des rôles</message>
-	<message key="admin.mergeUsers.confirm">Êtes-vous certain de vouloir fusionner le compte portant le nom d'utilisateur "{$oldUsername}" avec le compte portant le nom d'utilisateur "{$newUsername}"? Le compte portant le nom d'utilisateur "{$oldUsername}" sera supprimé. Cette action n'est pas réversible.</message>
+	<message key="admin.mergeUsers.confirm">Êtes-vous certain de vouloir fusionner le compte portant le nom d'utilisateur "{oldAccountCount}" avec le compte portant le nom d'utilisateur "{$newUsername}"? Le compte portant le nom d'utilisateur "{$oldAccountCount}" sera supprimé. Cette action n'est pas réversible.</message>
```

Comment:
I have compare the file that I have in my instance and the locale/fr_CA/admin.xml file on the master branch and I'm pretty sure that I have the latest version (master). Note that I have added the dollar sign that was missing on the oldAccountCount variable.
I will check all files to be sure that I have used the latest version from the master branch.